### DISCUSSION TITLE
Gpt plugin for easily generate blog posts and content?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [payloadcms/payload#3175] [Q&A]
I would like to contribute to or write a gpt plugin that can help in assisting writing marketing content mainly blog posts for my projects.

there are lots of prompts and workflows with gpt to write blog posts and i would like to build a plugin or contribute to an existing to “ capture” a good gpt prompt workflow for generating blog posts titles and content and implement good seo.

i do have some typescript code from an ecisting project that generates content this could be inproved upon.

gpt is hot ofcourse, so perhaps a core plugin would be an awesome way for payload to add an innovwtive solution for assisting blog post writing.

im not sure how it should actually look perhaos more like a 3 step wizard in payload  for 

1. Generate outline and blog entries with good titles annd content plus seo?
2. then be able to approve entries with a button click upon which the blog entry would be written to the detabase or markdown

**Comment by matthijs166** [comment]
Found this today: https://github.com/mycast-ai/payload-plugin-ai-completions

Looks like it can use some help ;)
